### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by jrwrigh** [numirias/qtile-plasma#30]
@numirias Are you still interested in maintaining this package? If not, I'd be willing to try and keep it going. There's still quite a few people who like to use this layout and the current version here is now broken for newer Python versions (see #29 #28).

**Comment by mhaffner** [comment]
@jrwrigh Perhaps it's time for a fork? It'd be nice to see this package maintained, but it looks like the owner might be incommunicado.

**Comment by jrwrigh** [comment]
Yeah, I think so. It'll be a bit before I have time to look into doing that. Feel free to go ahead if you want @mhaffner.
